src/language/typescript/2.0/serializers/operation-object.ts

```diff
@@ -411,8 +411,8 @@ export const serializeQueryParameterObject = (
 		case 'number':
 		case 'boolean': {
 			const f = serializedFragment(
-				`value => encodeURIComponent('${parameter.name}') + '=' + encodeURIComponent(value)`,
-				[],
+				`value => some(encodeURIComponent('${parameter.name}') + '=' + encodeURIComponent(value))`,
```

Comment:
I see that you add `some` in all `switch` branches, would it make more sense to keep this function simpler and operate on simple types and wrap its result in `some` on the call site?